main.tf

```diff
@@ -102,7 +102,7 @@ resource "aws_s3_bucket" "default" {
 }
 
 module "s3_user" {
-  source       = "git::https://github.com/cloudposse/terraform-aws-iam-s3-user.git?ref=tags/0.9.0"
+  source       = "git::https://github.com/cloudposse/terraform-aws-iam-s3-user.git?ref=tags/010.0"
```

Comment:
this is speculative based on the latest version of the module and https://github.com/cloudposse/terraform-aws-iam-s3-user/pull/19